snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -176,36 +170,34 @@ public static void AnyEx1()
             }
 
             // <Snippet130>
-            class Pet
+            class Pet(string name, int age)
             {
-                public string Name { get; set; }
-                public int Age { get; set; }
+                public string Name { get; set; } = name;
+                public int Age { get; set; } = age;
             }
-            class Person
+            class Person(string lastName, Pet[] pets)
             {
-                public string LastName { get; set; }
-                public Pet[] Pets { get; set; }
+                public string LastName { get; set; } = lastName;
+                public Pet[] Pets { get; set; } = pets;
             }
 
             public static void AnyEx2()
             {
-                List<Person> people = new List<Person>
-                    { new Person { LastName = "Haas",
-                                   Pets = new Pet[] { new Pet { Name="Barley", Age=10 },
-                                                      new Pet { Name="Boots", Age=14 },
-                                                      new Pet { Name="Whiskers", Age=6 }}},
-                      new Person { LastName = "Fakhouri",
-                                   Pets = new Pet[] { new Pet { Name = "Snowball", Age = 1}}},
-                      new Person { LastName = "Antebi",
-                                   Pets = new Pet[] { }},
-                      new Person { LastName = "Philips",
-                                   Pets = new Pet[] { new Pet { Name = "Sweetie", Age = 2},
-                                                      new Pet { Name = "Rover", Age = 13}} }
-                    };
+                List<Person> people = [
+                    new ("Haas", [
+                        new ("Barley", 10 ),
+                        new ("Boots", 14 ),
+                        new ("Whiskers", 6 ) ]),
+                    new ("Fakhouri", [ new ("Snowball",  1) ]),
+                    new ("Antebi", []),
+                    new ("Philips", [
+                        new ("Sweetie",  2),
+                        new ("Rover",  13) ] )
+                    ];
 
                 // Determine which people have a non-empty Pet array.
                 IEnumerable<string> names = from person in people
-                                            where person.Pets.Any()
+                                            where person.Pets.Length != 0

```

Comment:
Given that these are snippets specifically intended to showcase use of Linq why not just keep `Any`?

Reply:
I'd prefer `Any` here as well.